Fix UI issues in login and signup pages

j-mahapatra opened the pull request:
# Description

- It fixes the improper alignment of the UI of the OAuth buttons in login page.
- It converts CSS styles into tailwindCSS in login and signup pages.

Fixes #137 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Responsive Testing?
(Only applicable for Frontend changes)
Checked for: (width in px)
- [x] Laptop L - 1440px
- [x] Laptop - 1024px
- [x] Tablet - 768px
- [x] Mobile M - 375px

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in modules

# Screenshots:

![screenshot1](https://github.com/digitomize/digitomize/assets/107102771/2ec74abc-6522-402d-be4d-d6766b861813)
![screenshot2](https://github.com/digitomize/digitomize/assets/107102771/8a272795-05d0-456f-94c6-34af00296802)
![Screenshot3](https://github.com/digitomize/digitomize/assets/107102771/46d80b6d-7efb-4463-8538-f356c11b711e)
![Screenshot4](https://github.com/digitomize/digitomize/assets/107102771/7941b71b-d3e9-4381-8fa8-be6d83b67eb9)


# Note for Reviewer:

The Login button is misaligned and needs to be centered. The OAuth buttons are centered according to the container.

pranshugupta54 commented:
Hey, the issue was to fix the UI. Since you've aligned it better, can you also fix the "Login" button's alignment and the "new user signup" text too

j-mahapatra commented:
I made the necessary changes. I updated the screenshots as well. I also fixed another issue in login page in the last commit where the heading text was being overlapped by navbar. Please take a look.

pranshugupta54 commented:
Hey @j-mahapatra, can you also convert the page into tailwind? Like remove the css file for this

j-mahapatra commented:
Can you create a separate issue for converting the CSS into tailwind? You can assign me to it.

pranshugupta54 commented:
Looks like this messed up the [Signup Page](https://deploy-preview-139--v2-digitomize.netlify.app/signup)
<img width="1439" alt="image" src="https://github.com/digitomize/digitomize/assets/76090263/52f2200d-4d8a-480a-9ae5-954b15911991">

pranshugupta54 commented:
> Can you create a separate issue for converting the CSS into tailwind? You can assign me to it.

sure

j-mahapatra commented:
Yes, the signup page is also using the 'login-container' class. I removed it on instinct because of its name. I will fix it in a minute.

pranshugupta54 commented:
> Can you create a separate issue for converting the CSS into tailwind? You can assign me to it.

#140

pranshugupta54 commented:
Join us on [Discord](https://discord.gg/bsbBytBqBc) for better discussion.

inbuilt-aura commented:
@pranshugupta54 i am working on this,will update soon..

pranshugupta54 commented:
Hey @j-mahapatra, did you fix it?

j-mahapatra commented:
I was caught up with something. I will push the final commit soon.

j-mahapatra commented:
Hi @pranshugupta54, I made the changes as required and updated the PR description with new screenshots. Please take a look.

pranshugupta54 commented:
Hey @j-mahapatra, we're getting a small right gap in this (on Login Page, works fine for signup)

<img width="350" alt="image" src="https://github.com/digitomize/digitomize/assets/76090263/3f7c0e9f-1e45-4add-ab38-be57cc0151f6">

j-mahapatra commented:
I made a small change to manage overflows in the login container which is fixing the above issue. Please take a look.

pranshugupta54 commented:
You can also use same design for Signup Button like Login

pranshugupta54 commented:
Also add a line like "Already registered? Login"

pranshugupta54 commented:
{error} div needs some fix
When there's an error to authenticate, the div is changing alignment too
<img width="1440" alt="image" src="https://github.com/digitomize/digitomize/assets/76090263/d44b3682-57bd-46e2-b7ed-fde84840dd07">

pranshugupta54 commented:
@j-mahapatra, we're getting a horizontal scroll on Mobile - S
Preview: https://deploy-preview-139--v2-digitomize.netlify.app/login

pranshugupta54 commented:
https://github.com/digitomize/digitomize/assets/76090263/7e206217-fe70-41f8-9e03-f5200df31677

pranshugupta54 commented:
- [ ] Input boxes are increasing in size


https://github.com/digitomize/digitomize/assets/76090263/369071ed-ce31-4661-8242-5ecb8a456894

j-mahapatra commented:
The input box size increase issue is not happening on my end. 

https://github.com/digitomize/digitomize/assets/107102771/500cec27-edbc-4099-b13d-ef591bc49fd3

pranshugupta54 commented:
I'm getting this in Brave and Chrome (in Mac)

pranshugupta54 commented:
Basically the input fields have w-full 🤔, so when the parent-parent div changes size due to error, it increases the width too.
Just try if you can fix this otherwise we'll merge it.

<img width="1166" alt="image" src="https://github.com/digitomize/digitomize/assets/76090263/a5cb272c-d3a5-4407-857e-7f3e150cf5db">

j-mahapatra commented:
I am still not able to replicate the above issue on my device. I understood why it is happening in the video that you have shared but it doesn't occur on my end for some reason. Should I just set a fixed width for the input containers?

pranshugupta54 commented:
Yes, you can set fixed widths (depending on screen size)

j-mahapatra commented:
I have made a simple change that should fix the issue. I am not able to test it because I am not getting the behavior that we are discussing. Please check it on your end once.

pranshugupta54 commented:
@j-mahapatra Perfect 🎉

pranshugupta54 commented:
Thank you so much for fixing this!

inbuilt-aura commented:
is this done ?

pranshugupta54 commented:
hey @inbuilt-aura, this is done and merged!

pranshugupta54 commented:
@all-contributors please add j-mahapatra for code
